cypress/e2e/signIn.cy.js

```diff
@@ -2,6 +2,15 @@
 
 describe('Sign In page', () => {
   it('should provide an ability to log in', () => {
+    cy.visit('https://conduit.mate.academy/');
+    cy.get(':nth-child(2) > .nav-link').click();
+    cy.url().should('contain', '/login');
 
+    cy.get(':nth-child(1) > .form-control').type('oaao@gmail.com');
+    cy.get(':nth-child(2) > .form-control').type('1234qaz');
```

Comment:
Hardcoding credentials in the test is not a good practice. It's better to use environment variables or other secure ways to handle credentials.